src/main/java/io/github/dsheirer/controller/channel/ChannelProcessingManager.java

```diff
@@ -308,6 +312,29 @@ private void startProcessing(ChannelStartProcessingRequest request) throws Chann
             mLog.debug("Error obtaining source for channel [" + channel.getName() + "]", se);
         }
 
+        if(ignoreMutedTalkgroups && request.hasIdentifierCollection())
+        {
+            Identifier identifier = request.getIdentifierCollection().getIdentifier(IdentifierClass.USER, Form.TALKGROUP, Role.TO);
+            if (identifier != null)
+            {
+                AliasList aliasList = mAliasModel.getAliasList(request.getIdentifierCollection());
+                List<Alias> aliasesFromTalkgroupIdentifier = aliasList.getAliases(identifier);
+
+                Optional<Alias> mutedAlias = aliasesFromTalkgroupIdentifier.stream().filter(alias -> alias.priorityProperty().getValue() < 0).findAny();
+                if (mutedAlias.isPresent()) {
+                    mLog.debug("Not processing channel due to talkgroup ID " + identifier + " because of muted alias " + mutedAlias.get() + " via " + aliasList.getName());
```

Comment:
Specify a format instead of string concat for these logging statements. 

`mLog.debug("Not processing channel due to talkgroup ID {} because of muted alias {}  via {}.", identifier, mutedAlias.get(), aliasList.getName());`

Doing String concat results in the runtime environment building the string every time, even when we're not debug logging. Passing it via format only builds the string when the logging level is active.